lib/concurrent/edge/future.rb

```diff
@@ -100,25 +128,26 @@ def initialize(promise, default_executor = :io)
         ensure_ivar_visibility!
       end
 
+      # @return [:pending, :completed]
       def state
-        @State.get
+        @State.get.to_sym
```

Comment:
see lines https://github.com/ruby-concurrency/concurrent-ruby/pull/314/files#diff-8a184cb7a6ea329fe4d978de8181531fR132 and https://github.com/ruby-concurrency/concurrent-ruby/pull/314/files#diff-8a184cb7a6ea329fe4d978de8181531fR366